res/css/views/rooms/_RoomHeader.pcss

```diff
@@ -14,6 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+:root {
```

Comment:
To be honest, I guess this is not exactly how custom properties are expected to be defined, using the logic of cascading. Ideally it should be:

1. Define variables on `_custom.pcss` (any one imported earlier than others) for indicators which should be applied globally, such as `--indicator-size`, `--indicator-offset` and  `--indicator-pulseColor`
2. Let them cascade and customize for each case

Setting up different custom properties for each case like below is "preprocessor-ly".